### ISSUE TITLE
clustertriggerbindings.tekton.dev "my-clustertriggerbinding" is not deleted in e2e

### ISSUE THREAD

**Issue by zhangtbj** [tektoncd/triggers#501]
The e2e tests work fine when I first executing on my cluster.

But when I run the e2e test second time, I got an error:
```
--- FAIL: TestEventListenerCreate (2.85s)
    eventlistener_test.go:66: Create namespace arrakis-hrkq2 to deploy to
    eventlistener_test.go:66: Verify SA default is created in namespace arrakis-hrkq2
    eventlistener_test.go:72: Start EventListener e2e test
    eventlistener_test.go:167: Error creating ClusterTriggerBinding: clustertriggerbindings.tekton.dev "my-clustertriggerbinding" already exists
```

It means the `clustertriggerbindings.tekton.dev "my-clustertriggerbinding"` doesn't be cleaned up after e2e test.

It need to be deleted. Thanks!

**Comment by ncskier** [comment]
Is this a duplicate of https://github.com/tektoncd/triggers/issues/495. Does this only happen when the first test fails? fyi @dibyom

**Comment by zhangtbj** [comment]
It happen from second test. there is no `my-clustertriggerbinding` at the first test.

**Comment by dibyom** [comment]
yeah this is a duplicate of #495 and should be fixed via #497

**Comment by dibyom** [comment]
Will close this in favor of #495 
/close
